test/onnx/test_fx_to_onnx.py

```diff
@@ -669,11 +669,12 @@ def forward(self, x):
                 return self.normal.sample(x.shape)
 
         x = torch.randn(2, 3)
-        exported_program = torch.export.export(Model(), args=(x,))
-        _ = torch.onnx.dynamo_export(
-            exported_program,
-            x,
-        )
+        with torch.no_grad():
```

Comment:
Explicitly add it because the defined Model invokes `troch.no_grad` under the hood and the dynamo produces the graph IR with `torch._C._set_grad_enabled` op which we intend not to allow in the later stage. Setting the grade is false before export can eliminate the `torch._C._set_grad_enabled` op in the graph IR, to make this test case pass.

Reply:
cc: @thiagocrepaldi